extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcDevModeConfig.java

```diff
@@ -8,6 +8,8 @@ public class ArcDevModeConfig {
 
     /**
      * If set to true then the container monitors business method invocations and fired events during the development mode.
+     * <p>
+     * NOTE: Changing this configuration requires full rebuild.
```

Comment:
Hm, I'd rather say something like _"this config property should not be changed in the development mode as it requires a full rebuild of the application"_... WDYM?

Reply:
I don't mind either. I was just trying to come up with something short and simple that will be noticeable from the dev ui.
How about something like this:
> NOTE: Changing this option requires full rebuild and therefore it should not be done directly from Dev UI

Reply:
The Dev UI is not enough. You could change a value in the `application.properties` file for example...

Reply:
Good point, that's what effectively happens if you change it via Dev UI anyway.

Reply:
I've changed to wording to what you suggested @mkouba